src/components/form/examples/span-fields-schema.ts

```diff
@@ -46,7 +46,7 @@ export const schema = {
                 },
                 layout: {
                     colSpan: 2,
-                    rowSpan: 3,
+                    rowSpan: 1,
```

Comment:
this shouldn't be a part of the change. 😄

Reply:
ah, I will remove it, of course 🤗